src/googleapis.js

```diff
@@ -1,12 +1,18 @@
 const { GoogleToken } = require('gtoken');
 const request = require('request');
 
-const getToken = keyFile => {
+const getToken = ({ keyFile, key }) => {
   return new Promise((resolve, reject) => {
-    const gtoken = new GoogleToken({
-      keyFile,
-      scope: ['https://www.googleapis.com/auth/drive'],
-    });
+    const gtoken = keyFile ?
+      new GoogleToken({
+        keyFile,
+        scope: ['https://www.googleapis.com/auth/drive']
```

Comment:
Could we move the `scope` in a constant so that we don't repeat the same value?

Reply:
done